code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -561,16 +561,16 @@
 	icon_state = "petcollar"
 	item_color = "petcollar"
 	var/tagname = null
-	var/obj/item/weapon/card/id/access_id
+	var/obj/item/weapon/card/id/id
```

Comment:
you really shouldn't be arbitrarily changing vars like this, just because you feel like it, unless there's an exceptional reason to be doing so.